app/lib/queues/populatePracticeQueue.js

```diff
@@ -12,16 +12,18 @@ const steps = ['overview', 'facilities', 'services'];
 const numberOfSteps = steps.length;
 let hitsPerWorker;
 let count = 0;
+let retryCount = 0;
```

Comment:
Can both `count` and `retryCount` be hidden from the top level of the module? It feels like it would be a bit too easy for either of them to be changed in ways that they were not originally intended to be changed.